.github/workflows/test-package.yml

```diff
@@ -106,7 +106,7 @@ jobs:
       env:
         DATABASE_ENGINE: ${{ matrix.database }}
       run: |
-        coverage run --append --source=admin_interface setup.py test
+        coverage run --append --source=admin_interface runtests
```

Comment:
CI is broken, this should be `coverage run --append --source=admin_interface runtests.py`

Reply:
```suggestion
        coverage run --append --source=admin_interface runtests.py
```